### PR TITLE
updated dependices

### PR DESCRIPTION
**Pull request by mikytron123**

### CODE REVIEW

**Comment by mikytron123** [comment]
This build passed the check, do I need to do anything else to add the new code?

**Comment by Gibstick** [comment]
You can get the code back by cherry-picking the commit: 
```sh
git cherry-pick 8e196215c6c3588ba5350d207a0da9a9a95d2df0
```

File: docker/Dockerfile [hunk 1]
```diff
@@ -18,7 +18,7 @@ COPY src ./src
 # FIXME: the data directory is specified in the project file and therefore gets
 # copied in a publish. This means that changes in the data changes trigger a
 # full rebuild.
-RUN dotnet publish -c release -o /app --no-restore
+RUN dotnet publish -c release -o /app --no-restore /restore
```
Comment (Gibstick) [inline docker/Dockerfile]: What does the `/restore` do here?

Comment (mikytron123) [reply docker/Dockerfile]: it gives this error if exclude it on my machine 1.185 /usr/share/dotnet/sdk/3.1.415/Sdks/Microsoft.NET.Sdk/targets/Microsoft.PackageDependencyResolution.targets(241,5): error MSB4018: The "ResolvePackageAssets" task failed unexpectedly. [/build/src/NadekoBot/NadekoBot.csproj]

Comment (Gibstick) [reply docker/Dockerfile]: If you change it to `RUN dotnet publish -c release -o /app` does it work for you?

Comment (mikytron123) [reply docker/Dockerfile]: yes that command works

Comment (Gibstick) [reply docker/Dockerfile]: This means the Dockerfile and/or the dependencies are still broken somehow. The `/restore` is actually undoing the `--no-restore` cause `/flag` is the Windows way of adding command line flags to things and I guess they carried it over to the dotnet command.

Comment (Gibstick) [reply docker/Dockerfile]: Are you saying the dockerfile fails to build on your machine or you need to run that command locally for it to work?

Comment (mikytron123) [reply docker/Dockerfile]: i mean dockerfile fails to build

**Comment by mikytron123** [comment]
The check passed so it should be ok now

**Comment by Gibstick** [comment]
The fact that we needed to get rid of `--no-restore` is a bit sus but it's ok for now

**Review by Gibstick — APPROVED** [review]

**Pull request merged**